### PR TITLE
Add school partnership form

### PR DESCRIPTION
**Pull request by ethanmills**
I'm going to talk to Dom and Shani about the copy on all of these, but I want to get them on Develop first so we have something to talk about.

Ticket: https://trello.com/c/7Jz1rMZR/49-prototype-declare-partnership-with-single-school-flow

### CODE REVIEW

File: app/views/declare-partnership/index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>
+        {% extends "layout.html" %}
```
Comment (kicferk1) [inline app/views/declare-partnership/index.html]: If we extend layout, do we need the first part of the file? Can we stick them in `layout.html` and forget about them?

File: app/views/declare-partnership/single/success.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        text: "Back",
+        href: "/declare-partnership"
+    }) }}
+{% endblock %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div style="  border: 4px solid #00703c;padding: 20px;margin-top: 0; margin-bottom: 40px">
```
Comment (kicferk1) [inline app/views/declare-partnership/single/success.html]: Let's make it into a class?

File: app/views/declare-partnership/single/success.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                <p class="govuk-heading-m">School added</p>
+                <p style="color: #00703c"><strong>Your partnership with {{ data["declare-partnership-school-name"] }}
```
Comment (kicferk1) [inline app/views/declare-partnership/single/success.html]: Is there a class making something `govuk green`? If not can we make one and use it? 

https://design-system.service.gov.uk/styles/colour/

File: app/views/declare-partnership/single/index.html [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    {{ govukBackLink({
+        text: "Back",
+        href: "/declare-partnership"
+    }) }}
+{% endblock %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-grid-row">
```
Comment (kicferk1) [inline app/views/declare-partnership/single/index.html]: A lot of grid rows around here. Do we need this one in particular?

File: app/views/declare-partnership/single/failed.html [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    {{ govukBackLink({
+        text: "Back",
+        href: "/declare-partnership"
+    }) }}
+{% endblock %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-grid-row">
```
Comment (kicferk1) [inline app/views/declare-partnership/single/failed.html]: Ditto, do we need this particular row?

File: app/views/declare-partnership/index.html (hunk 1 above)
Comment (ethanmills) [reply app/views/declare-partnership/index.html]: Nope, don't need these, thought I'd got rid of them. Good spot

**Review by kicferk1 — APPROVED** [review]

**Review by amazimbe — APPROVED** [review]

**Pull request merged**